cmd/main.go

```diff
@@ -51,10 +51,13 @@ func main() {
 						dataset, _ := mystery.LoadDataset(mystery.SwiftLang)
 						ingredients := dataset.Generate("", flagImports)
 						mystery.PrintGenerate("Swift", ingredients)
+					case mystery.KotlinLang:
+						dataset, _ := mystery.LoadDataset(mystery.KotlinLang)
```

Comment:
Can you re-commit with gofmt here as well? Spacing is off

Reply:
sure